oresat_dxwifi/templates/oresat_live.html

```diff
@@ -26,7 +26,7 @@ <h2>Write</h2>
         </div>
     </body>
     <script>
-        const STATE_INDEX = "live_status"
+        const STATE_INDEX = "status"
```

Comment:
If we change it here, we need to change it in `oresat_dxwifi/services/oresat_live.py`.

Reply:
Should already be changed

Reply:
yeah must've been a later commit that I haven't gotten to yet